src/main/java/Ava/Exceptions/CommandNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package Ava.Exceptions;
+
+public class CommandNotFoundException extends AvaException {
+    private final String ERROR_PROMPT = "Command Not Complete or Possible !!";
```

Comment:
The other Exception classes seem to give a clear indicator of what the correct way is for the user to input text.
Perhaps this Exception could tell the user the correct command formats?